itest/server_harness.go

```diff
@@ -74,7 +74,7 @@ func (s *serverHarness) start() error {
 	)
 
 	mailboxServer, err := mailbox.NewServer(
-		s.serverHost, connData, grpc.WithTransportCredentials(
+		s.serverHost, connData, nil, grpc.WithTransportCredentials(
```

Comment:
Can we add a check to some of our LNC itests that cover these new status callbacks?